samples/matlab_new/reactor2.m

```diff
@@ -0,0 +1,60 @@
+function reactor2(g)
```

Comment:
Broken
```
>> reactor2
  reactor2 Zero-dimensional kinetics: adiabatic, constant volume.
 
     This example illustrates how to use class 'Reactor' for
     zero-dimensional kinetics simulations. Here the parameters are
     set so that the reactor is adiabatic and constant volume.

Error using calllib
Method was not found.

Error in Reactor (line 55)
            r.id = calllib(ct, 'reactor_new2', typ);

Error in IdealGasReactor (line 12)
    r = Reactor(contents, 'IdealGasReactor');

Error in reactor2 (line 20)
    r = IdealGasReactor(gas);

```